ert_gui/simulation/run_dialog.py

```diff
@@ -191,6 +194,36 @@ def on_new_iteration(self, parent: QModelIndex, start: int, end: int) -> None:
 
             self._tab_widget.addTab(widget, f"Iteration {iter}")
 
+    @Slot(QModelIndex)
+    def _job_clicked(self, index):
+        node = index.internalPointer()
+        if node is None or node.type != NodeType.JOB:
+            return
+
+        parent = node.parent
+        while parent.type != NodeType.ROOT:
+            if parent.type == NodeType.REAL:
+                real = parent.id
+            elif parent.type == NodeType.ITER:
+                iter = parent.id
```

Comment:
All of this data resides in the model. You could expose that such that you can do

`index.model().get_iter()` and `index.model().get_real()`?

I think that's OK.